### PR TITLE
2116 retrieve admin and moderators

### LINKED ISSUE #2116: 🚀 [Feature] retrieve all admins and moderators.
## 🚀 Feature
retrieve all admins and moderators. 

there should be an api call that makes it possible to list all admins and moderators with their names in the frontend. 
these names appear on the info page and this serves the purpose of transparency in the community.

### PR DESCRIPTION
**Pull request by Elweyn**
## 🍰 Pullrequest

As User we want to see the moderators and admins of the community

### Issues

- fixes #2116 

### Todo

- [X] Implement resolver method to query moderators and admins

### CODE REVIEW

File: backend/src/graphql/resolver/UserResolver.ts [hunk 1]
```diff
@@ -734,6 +738,46 @@ export class UserResolver {
...
+      skip: (currentPage - 1) * pageSize,
+      take: pageSize,
+    })
+
+    if (users.length === 0) {
+      return {
+        userCount: 0,
+        userList: [],
+      }
+    }
```
Comment (Mogge) [inline backend/src/graphql/resolver/UserResolver.ts:766]: I think this is not needed.

File: backend/src/graphql/resolver/UserResolver.ts [hunk 2]
```diff
@@ -734,6 +738,46 @@ export class UserResolver {
...
+        return {
+          firstName: user.firstName,
+          lastName: user.lastName,
+        }
+      }),
+    )
+    return {
+      userCount: count,
+      userList: adminUsers,
+    }
```
Comment (Mogge) [inline backend/src/graphql/resolver/UserResolver.ts:779]:
```suggestion
    return {
      userCount: count,
      userList: users.map((user) => {
        return {
          firstName: user.firstName,
          lastName: user.lastName,
        },
    }
```

File: backend/src/graphql/resolver/UserResolver.ts [hunk 3]
```diff
@@ -734,6 +738,46 @@ export class UserResolver {
...
+
+    const adminUsers = await Promise.all(
+      users.map((user) => {
+        return {
+          firstName: user.firstName,
+          lastName: user.lastName,
+        }
+      }),
+    )
```
Comment (Mogge) [inline backend/src/graphql/resolver/UserResolver.ts:775]: Not needed

**Review by Mogge — CHANGES_REQUESTED** [review]
What about a simple test?

**Comment by ogerly** [comment]
this PR is part noted in #1973 , works and goes !

**Review by Mogge — APPROVED** [review]

**Pull request merged**